biofefi/machine_learning/learner.py

```diff
@@ -53,7 +54,13 @@ def _fit_holdout(self, data: Tuple) -> None:
             for model_name, model in self._models.items():
                 res[i][model_name] = {}
                 self._logger.info(f"Fitting {model_name} for bootstrap sample {i+1}...")
-                model.fit(X_train, y_train)
+
+                if model_name == ModelNames.BRNN:
```

Comment:
Rather than testing the model name, check the type of `X_train` and `y_train`. If they're not numpy arrays like expected, convert them with `to_numpy` like you're doing here.

Reply:
Ok Dan